pkg/metadata/inventories/inventories.go

```diff
@@ -103,6 +103,16 @@ const (
 	AgentAPMEnabled                     AgentMetadataName = "feature_apm_enabled"
 	AgentIMDSv2Enabled                  AgentMetadataName = "feature_imdsv2_enabled"
 
+	// System Probe general config values
+	AgentSPOOMKillEnabled               AgentMetadataName = "feature_oom_kill_enabled"
+	AgentSPTCPQueueLengthEnabled        AgentMetadataName = "feature_tcp_queue_length_enabled"
+	AgentSPTelemetryEnabled             AgentMetadataName = "system_probe_telemetry_enabled"
+	AgentSPCOREEnabled                  AgentMetadataName = "system_probe_core_enabled"
+	AgentSPRuntimeCompilationEnabled    AgentMetadataName = "system_probe_runtime_compilation_enabled"
+	AgentSPKernelHeadersDownloadEnabled AgentMetadataName = "system_probe_kernel_headers_download_enabled"
+	AgentSPPrebuiltFallbackEnabled      AgentMetadataName = "system_probe_prebuilt_fallback_enabled"
+	AgentSPMaxConnectionPerMessage      AgentMetadataName = "system_probe_max_connections_per_message"
```

Comment:
don't you want to keep `feature_` prefix?

Reply:
looking at the existing content of this file, it looks that only configuration of the actual product functionality have this prefix.